### ISSUE TITLE
Reproducing benchmark issue

### ISSUE THREAD

**Issue by cneyang** [thuml/Transfer-Learning-Library#111]
**Describe the bug**
Cannot reproduce benchmark result reported in [benchmarks](https://github.com/thuml/Transfer-Learning-Library/blob/master/docs/dalib/benchmarks/image_classification.rst).
I trained DANN on Office31 Amazon -> Webcam task using ResNet 50 with shell script in `example` directory as below.
```
CUDA_VISIBLE_DEVICES=0 python dann.py data/office31 -d Office31 -s A -t W -a resnet50 --epochs 20 --seed 1 --log logs/dann/Office31_A2W
```
I tried with different seeds (1 ~ 4) but the best accuracy is about 80%, which is different with the reported accuracy of 91.4%.

**Hardware and Software that are used**
OS: Ubuntu 20.04.2 LTS
PyTorch: 1.7.1
torchvision: 0.8.2
GPU: RTX 2080Ti

**Comment by thucbx99** [comment]
Thank you for using the recommended format for submitting an issue. We will verify and resolve this issue soon.

**Comment by thucbx99** [comment]
I used the same shell script and environment as you for training. After 5000 iterations, the top-1 accuracy reached 91.7%, matching the results we reported. Please check if you are using the latest version of our library.
